mock-core-configs/etc/mock/templates/fedora-rawhide.tpl

```diff
@@ -42,7 +42,7 @@ user_agent={{ user_agent }}
 
 {%- macro rawhide_gpg_keys() -%}
 file:///usr/share/distribution-gpg-keys/fedora/RPM-GPG-KEY-fedora-$releasever-primary
-{%- for version in [releasever|int, releasever|int - 1]
+{%- for version in [releasever|int, releasever|int - 1, releasever|int + 1]
```

Comment:
Thanks! My only qualm is what happens if the key doesn't exist yet? We don't want any kind of warning or error in that case. Hmm, I think we get an error if they key doesn't exist:
```
cannot open file: (2) - No such file or directory [/usr/share/distribution-gpg-keys/fedora/RPM-GPG-KEY-fedora-46-primary]
```

I think this should be wrapped in a check whether the key actually exists.

Reply:
Good point, I installed this fatal error into the branching script:

```
File /usr/share/distribution-gpg-keys/fedora/RPM-GPG-KEY-fedora-43-primary not found!

The new Fedora Rawhide (42) requires a GPG key for Fedora 43,
which must be generated by Fedora Infrastructure team and shipped in the
distribution-gpg-keys.rpm package.  This either has not happened yet, the
package is outdated, or it is not even installed on this box.
```